README.md

```diff
@@ -35,21 +45,30 @@ To set it up, clone the repo, making sure to checkout the tag corresponding to t
 
     git clone git@github.com:digital-asset/ex-tutorial-nodejs.git
     cd ex-tutorial-nodejs
-    git checkout 0.5.0
+    git checkout v0.6.0
 
 The repo includes `daml/PingPong.daml`, which is the source for a DAML module with two templates: `Ping` and `Pong`. The app uses these.
 
 [Back to the table of contents](#table-of-contents)
 
+## Compile the DAML code
+
+Before getting started, build the DAML code with
+
+    daml build
+
+[Back to the table of contents](#table-of-contents)
+
 ## Run the sandbox
 
 Use the sandbox to run and test your application.
 
-Now that you created the project and you're in its directory, start the sandbox by running:
+Now that you created the project and you're in its directory:
```

Comment:
This phrasing seems odd, since the previous step already needed to happen within the project directory.

Reply:
Addressed by https://github.com/digital-asset/ex-tutorial-nodejs/pull/7/commits/482a5ab5f421d587c5258115ef755a98863df47b